typescript/infra/scripts/interchain/deploy.ts

```diff
@@ -0,0 +1,100 @@
+import { Wallet } from 'ethers';
```

Comment:
Some comments about how this script differs from similar deploy scripts would be helpful here

Reply:
for sure